src/main/java/org/apache/commons/lang3/StringUtils.java

```diff
@@ -9245,25 +9245,28 @@ public static String unwrap(final String str, final char wrapChar) {
             index += Character.charCount(result[i]);
         }
         return result;
-    }    
+    }
     
     /**
      * <p>Finds index of all the occurences of given search key found in source string.
      * </p>
-     * @param source
-     * @param searchKey
+     * @param source 
+     *           input string to find indexes
+     * @param searchKey 
+     *           search character
      * @return list of integer of indexes.
      */
     public static List<Integer> indexesOf(final CharSequence source, final Character searchKey) {
     	if(isEmpty(source) || searchKey == null ) {
-    	    return null;
-    	}
+			return null;
+		}
```

Comment:
Can you fix the formatting, please?

Reply:
Ha-ha，They refused to merge only two lines of code modification